WcaOnRails/db/seeds/teams.seeds.rb

```diff
@@ -0,0 +1,9 @@
+if Team.count == 0
```

Comment:
Why wrap this in an `if Team.count == 0`? Can we unconditionally do this when seeding?

nit: This file would be a bit cleaner if you used ActiveRecord or FactoryGirl here instead, as you wouldn't need to set the `created_at` and `updated_at` fields here.